### PR TITLE
feat/MSSDK-1334: remove redundant aria attributes; move some stuff to typescript

### PR DESCRIPTION
**Pull request by Saddage**
### Description

Removed some unnecessary aria attributes.
While I was at it, I also moved some code from JavaScript to TypeScript.

### CODE REVIEW

File: src/components/Capture/Capture.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React, { useEffect, useState } from 'react';
+import { useEffect, useState } from 'react';
```
Comment (m4tewoosh) [inline src/components/Capture/Capture.tsx:1]: We need to remember later, when we make react-16 release to add this import on branch with 16

Comment (m4tewoosh) [reply src/components/Capture/Capture.tsx:1]: And in other places where this import is removed

File: src/components/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            <CheckMarkStyled
+              data-testid="checkmark"
+              $isMyAccount={isMyAccount}
+              $isRadioButton={isRadioButton}
+            />
+          )}
+        </CheckFrameStyled>
+        <ConsentDefinitionStyled
+          dangerouslySetInnerHTML={{
+            __html: `${children}${required && isMyAccount ? '*' : ''}`
```
Comment (m4tewoosh) [inline src/components/Checkbox/Checkbox.tsx:87]: Maybe we could pull this out and make variable with some meaning name, it would be easier to read

**Review by m4tewoosh — DISMISSED** [review]

**Comment by m4tewoosh** [comment]
Good job!  🚀

**Review by pawelacio — APPROVED** [review]

**Pull request merged**